chocolatey/tools-offline/chocolateyinstall.ps1

```diff
@@ -21,3 +21,11 @@ $packageArgs = @{
   validExitCodes= @(0, 3010, 1641)
 }
 Install-ChocolateyInstallPackage @packageArgs
+
+$installInfo = @"
+tool: chocolatey
```

Comment:
I think we expect these three fields to not be at the top level, but within a `install_method` field. 

Using this as a reference: https://github.com/DataDog/datadog-agent/pull/5414/files#diff-ee095aa3ebc9c402027d28eef39fd88bR349

@kbogtob to confirm.

Reply:
I think you are right (waiting on @kbogtob to confirm), I misread the Jira card.